src/pages/Lend/LoanChannelCategoryControl.vue

```diff
@@ -339,62 +347,61 @@ export default {
 		preFetch(config, client, args) {
 			return client.query({
 				query: loanChannelPageQuery
-			}).then(({ data }) => {
-				// filter routes on route.param.category to get current path
+			}).then(async ({ data }) => {
+				// Filter routes on route.param.category to get current path
 				const targetedLoanChannelURL = _get(args, 'route.params.category');
-				// isolate targeted loan channel id
+
+				// Isolate targeted loan channel id
 				const targetedLoanChannelID = getTargetedChannel(targetedLoanChannelURL, data);
-				// extract query
+
+				// Extract query
 				const pageQuery = _get(args, 'route.query');
 
-				return client.query({
-					query: loanChannelQuery,
-					variables: _merge(
-						{
-							ids: [targetedLoanChannelID],
-							limit: loansPerPage
-						},
-						fromUrlParams(pageQuery)
-					)
-				});
+				await preFetchChannel(
```

Comment:
Return isn't needed because of the `await`, but for consistency with the other chained promises in this `preFetch`, I removed the `await` and returned `preFetchChannel`.